db/snapshot/backup_task.cc

```diff
@@ -40,7 +40,7 @@ backup_task_impl::backup_task_impl(tasks::task_manager::module_ptr module,
     , _prefix(std::move(prefix))
     , _snapshot_dir(std::move(snapshot_dir))
     , _remove_on_uploaded(move_files) {
-    _status.progress_units = "bytes ('total' may grow along the way)";
```

Comment:
But it's still true, the "total" grows as the task runs, just much faster than it did before the change

Reply:
I agree it's not set when the task is created but rather when it starts running,
but once it's set to a non-zero value it doesn't grow anymore
I'm not sure sure it merits the disclaimer that it may grow along the way.
That's would be true for all tasks.  I don't think we ever commit on total being constant throughout tasks lifetime.

Reply:
Cc @Deexie

Reply:
Right. We can accumulate the total in a variable and set it once on success

Reply:
In v9 ([1ab3ec0](https://github.com/scylladb/scylladb/commit/1ab3ec061b206aa11dfc08155bb217ed2d7e594e)): calculated using a variable and assigned once to progress.total